cara/apps/calculator/__init__.py

```diff
@@ -211,7 +245,7 @@ def get(self):
 def make_app(
         debug: bool = False,
         calculator_prefix: str = '/calculator',
-        theme_dir: typing.Optional[Path] = None,
+        theme_dir: typing.Optional[Path] = None
```

Comment:
Would you mind addding this trailing comma back please?

Reply:
Sure, sorry for the accident.